test/manifest.ts

```diff
@@ -1750,6 +1825,86 @@ describe('Manifest', () => {
         ],
       ]);
     });
+
+    it('releases library in root (".")', async () => {
+      const manifest = JSON.stringify({
+        '.': '3.2.1',
+      });
+      const config = JSON.stringify({
+        packages: {
+          '.': {},
+        },
+      });
+
+      const github = new GitHub({
+        owner: 'fake',
+        repo: 'repo',
+        defaultBranch,
+      });
+      const mock = mockGithub(github);
+      expectManifest(mock, {manifest, lastReleaseSha});
+      expectPR(mock, {
+        lastReleaseSha,
+        mergedPRFiles: [
+          // lack of any "node/pkg2/ files indicates that package did not
+          // change in the last merged PR.
+          'package.json',
+          'CHANGELOG.md',
+        ],
+      });
+      expectGetFiles(mock, {
+        fixtureFiles: ['package.json'],
+        inlineFiles: [
+          ['release-please-config.json', config],
+          ['.release-please-manifest.json', manifest],
+          ['CHANGELOG.md', '#Changelog\n\n## v3.2.1\n\n* entry'],
+        ],
+      });
+      expectLabelAndComment(mock, {
+        addLabel: 'autorelease: tagged',
+        removeLabel: 'autorelease: pending',
+        prComments: [
+          ':robot: Release for googleapis is at https://pkg1@3.2.1:html :sunflower:',
```

Comment:
```suggestion
          ':robot: Release for googleapis is at https://googleapis@3.2.1:html :sunflower:',
```

this value comes from the mock `.resolves()` values on L1885-6 where I'll leave suggested changes too. It's just cosmetic but slightly confusing.